README.md

```diff
@@ -15,10 +15,10 @@ Dapr SDK for .NET allows you to implement the Virtual Actor model, based on the
 This repo builds the following packages:
 
 - Dapr.Client
-- Dapr.Client.Grpc
 - Dapr.AspNetCore
 - Dapr.Actors
 - Dapr.Actors.AspNetCore
+- Dapr.Microsoft.Extensions.Configuration.DaprSecretStore
```

Comment:
Can the package be called Dapr.Configuration.Extensions as the package contains other Configurations as well.

Reply:
Done